### ISSUE TITLE
Download percentage and remaining time completely off

### ISSUE THREAD

**Issue by michael-rapp** [flatpak/flatpak#3429]
## Linux distribution and version

Arch Linux

## Flatpak version

1.6.2

## Description of the problem

When downloading an update for a Flatpak package (org.gimp.GIMP in this case) via the terminal, I encountered a (minor) issue that caused the displayed download percentage, as well as the estimated remaining time, to take very extreme and unrealistic values. While downloading, the percentage grew far beyond 100% and thousands of hours were reported as the estimated download time (see attached screenshot). As this issue does usually not occur, it might have something to do with the download speed, which was very slow at the time (78,5 kB/s for downloading 110 MB).

![Screenshot from 2020-02-25 21-20-03](https://user-images.githubusercontent.com/6638695/75285429-e3a53000-5816-11ea-9b80-d468e2fde37e.png)

## Steps to reproduce

Run `$ flatpak update`

**Comment by dustContributor** [comment]
I have the exact same problem in Debian, with Flatpak version 1.6.2.

Moreover, mine gets stuck for a lot of time on the "Updating..." message (couple of minutes), before starting the download and showing the progress bar.

**Comment by mwleeds** [comment]
duplicate of #3448 (and that one has more info now)
